CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changelog
 --------------
 
 - **Enhancement:** Chunk field initializers and constructor statements across multiple init functions to avoid `MethodTooLargeException` in large graphs.
+- **Enhancement:** Add a new Metro option `warnOnInjectAnnotationPlacement` to disable suggestion to lift @Inject to class when there is only one constructor.
```

Comment:
I think enhancement entry should also mention that new check works for classes with any amount of params, not only for classes with an empty constructor

Reply:
Adjusted a bit